web/themes/joinup/prototype/sass/layouts/_layout.sass

```diff
@@ -10,27 +10,12 @@ div
 div
   box-sizing: border-box
 
-.main-container,
-.footer-container
-  box-sizing: border-box
+.main-container
   width: 100%
+  padding-left: 16px
```

Comment:
Additional padding is necessary on `main-container` class, without it, on smaller screens, items in the grid would touch the edge of the screen.